commands/operator-sdk/cmd/up/local.go

```diff
@@ -159,11 +153,13 @@ func upLocalAnsible() {
 
 func upLocalHelm() {
 	// Set the kubeconfig that the manager will be able to grab
-	if err := os.Setenv(k8sutil.KubeConfigEnvVar, kubeConfig); err != nil {
-		log.Fatalf("Failed to set %s environment variable: (%v)", k8sutil.KubeConfigEnvVar, err)
+	// only set env var if user explicitly specified a kubeconfig path
+	if kubeConfig != "" {
+		if err := os.Setenv(k8sutil.KubeConfigEnvVar, kubeConfig); err != nil {
+			log.Fatalf("failed to set %s environment variable: (%v)", k8sutil.KubeConfigEnvVar, err)
```

Comment:
Should we backport the adjusting of logs PR, to this branch as well? Otherwise, we will always have the same problem with CI. 🤔

Reply:
I don't think it would make sense to backport that. I feel like that's too big of a change to include in a micro/patch release. We'll only have issues with logging in the commands with cherry-picks, but it's pretty easy to manual fix those problems.